app/src/main/java/ingsw/group1/findmyphone/contacts/ContactConverter.java

```diff
@@ -0,0 +1,69 @@
+package ingsw.group1.findmyphone.contacts;
+
+import androidx.room.TypeConverter;
+
+import ingsw.group1.msglibrary.SMSPeer;
+
+/**
+ * Class of static methods necessary to convert {@link SMSPeer} to {@link Contact} and vice versa.
+ * It doesn't check the validity of a contact or peer.
+ *
+ * @author Giorgia Bortoletti
+ */
+class ContactConverter {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ContactConverter' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929954)